src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -968,6 +979,18 @@ private IEnumerable<FullBackup.StorageEnvironmentInformation> GetAllStoragesForB
                         // end of values
 
                         writer.WriteEndObject();
+                        writer.Flush();
+                        switch (outputStream)
+                        {
+                            case EncryptingXChaCha20Poly1305Stream encryptedStream:
+                                encryptedStream.Flush(flushToDisk: true);
+                                break;
+                            case Stream fileStream:
```

Comment:
Same here, `FileStream` handling

Reply:
FileStream is a stream. changed the name in https://github.com/ravendb/ravendb/pull/8506